### PR TITLE
Create run function with deserialized params

### PR DESCRIPTION
**Pull request by leeweisberger**
Create `run` function that we will call from the task shim (instead of basefunc) that deserializes params before passing them in through the task.

In a follow up, we will change the shim to call the `run` function.

cc @ericliuche

### CODE REVIEW

File: packages/airplane/internal/config.ts [hunk 1]
```diff
@@ -365,14 +374,33 @@ export const workflow = <TParams extends InputParams, TOutput>(
     >;
   };
 
+  const run = getRunFunc(f, config.parameters);
+
   wrappedF.__airplane = {
     type: "workflow",
     config: config,
     baseFunc: f,
+    run,
```
Comment (colinking) [inline packages/airplane/internal/config.ts:383]: The naming here seems quite confusing -- wrapped function (what a user calls to execute an inline task; expects InputParamValues), baseFunc (the old shim entrypoint; expects SerializedParamValues); run (the new shim entrypoint; expects SerializedParamValues).

Can we export `run` as `baseFunc` instead?

**Review by colinking — APPROVED** [review]

File: packages/airplane/internal/config.ts (hunk 1 above)
Comment (leeweisberger) [reply packages/airplane/internal/config.ts:383]: Two reasons I did this
1. For backwards compat. If you upgrade the sdk version but aren't on the latest cli, your task will start breaking. I don't really want to break more than we have to or try to time this perfectly.
2. It's the same convention as python https://github.com/airplanedev/lib/blob/main/pkg/build/universal-python-shim.py#L41

Comment (leeweisberger) [reply packages/airplane/internal/config.ts:383]: Doh nevermind I was thinking about why I left baseFunc around...

Ok I can update it we'll just be inconsistent with python and node. This way i don't have to change the shim

Comment (colinking) [reply packages/airplane/internal/config.ts]: Great! We should update Python at some point, too. cc @ericliuche

**Pull request merged**